### PR TITLE
Add Bitcoin Magazine podcast on Taproot activation

### PR DESCRIPTION
**Pull request by michaelfolkson**

### CODE REVIEW

**Comment by michaelfolkson** [comment]
Sorry didn't do `.md` rather than `.mdwn` Will revise and force push

File: bitcoin-magazine/2021-04-23-taproot-activation-update.md [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+
+Name: Sjors Provoost and Aaron van Wirdum
+
+Topic: Taproot Activation Update: Speedy Trial and the LOT=true client
+
+Location: Bitcoin Magazine (online)
+
+Date: April 23rd 2021
+
+Video: https://www.youtube.com/watch?v=SHmEXPvN6t4
```
Comment (adamjonas) [inline bitcoin-magazine/2021-04-23-taproot-activation-update.md:16]: nit: valid markdown would be to put this in brackets like `<https://www.youtube.com/watch?v=SHmEXPvN6t4>`

File: bitcoin-magazine/2021-04-23-taproot-activation-update.md [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+
+Previous episode on lockinontimeout (LOT): https://diyhpl.us/wiki/transcripts/bitcoin-magazine/2021-02-26-taproot-activation-lockinontimeout/
```
Comment (adamjonas) [inline bitcoin-magazine/2021-04-23-taproot-activation-update.md:18]: This should probably point to this repo: `<https://btctranscripts.com/bitcoin-magazine/2021-02-26-taproot-activation-lockinontimeout/>`

File: bitcoin-magazine/2021-04-23-taproot-activation-update.md [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+
+Previous episode on Speedy Trial: https://diyhpl.us/wiki/transcripts/bitcoin-magazine/2021-03-12-taproot-activation-speedy-trial/
```
Comment (adamjonas) [inline bitcoin-magazine/2021-04-23-taproot-activation-update.md:20]: `<https://btctranscripts.com/bitcoin-magazine/2021-03-12-taproot-activation-speedy-trial/>`

**Comment by michaelfolkson** [comment]
Addressed those nits @adamjonas and a couple more. I haven't squashed commits but I'm happy to (just unsure what is preferable on this repo).

**Comment by adamjonas** [comment]
I'll just squash them as I merge. Thanks!

**Pull request merged**